lib/core/presentation/widgets/poap/poap_claim_transfer_controller_widget/poap_claim_transfer_controller_widget.dart

```diff
@@ -76,17 +81,17 @@ class _PoapClaimTransferControllerWidgetViewState
         return PoapClaimedPopup(
           token: token,
           onClose: () {
-            _clear();
-            Navigator.of(context).pop();
+            clear();
```

Comment:
Should be more context for this
```suggestion
            clearData();
```